forge-gui/res/cardsfolder/upcoming/tricksters_elk.txt

```diff
@@ -0,0 +1,8 @@
+Name:Trickster's Elk
+ManaCost:1 G
+Types:Enchantment Creature Elk
+PT:3/3
+K:Bestow:1 G
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | SetPower$ 3 | SetToughness$ 3 | SetColor$ Green | RemoveAllAbilities$ True | AddType$ Creature & Elk | RemoveCardTypes$ True | RemoveCreatureTypes$ True | Description$ Enchanted creature loses all abilities and is a green Elk creature with base power and toughness 3/3. (It loses all other card types and creature types.)
+Oracle:Bestow {1}{G} (If you cast this card for its bestow cost, it's an Aura spell with enchant creature. It becomes a creature again if it's not attached to a creature.)\nEnchanted creature loses all abilities and is a green Elk creature with base power and toughness 3/3. (It loses all other card types and creature types.)

```

Comment:
we just use the exact print (for translation reason among other things)
so please remove extra reminder + empty line at end